Example/Example/Helpers/PostCell.swift

```diff
@@ -26,9 +26,9 @@ import UIKit
 
 class PostCell: UITableViewCell {
 
-    @IBOutlet weak var userImage: UIImageView!
-    @IBOutlet weak var postName: UILabel!
-    @IBOutlet weak var postText: UILabel!
+    @IBOutlet var userImage: UIImageView!
+    @IBOutlet var postName: UILabel!
+    @IBOutlet var postText: UILabel!
```

Comment:
Regarding making the IBOutlets strong, could you point me to documentation stating that weak is not recommended? I read the opposite in this [doc](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/LoadingResources/CocoaNibs/CocoaNibs.html#//apple_ref/doc/uid/10000051i-CH4-SW6).
Also connecting an Outlet from Xcode defines it as weak by default

Reply:
Apple documentation is confuse, I admit. So I remade the outlets from XCode itself to be sure

https://github.com/xmartlabs/XLPagerTabStrip/assets/1507391/9824393a-62fe-400e-ae30-943f87cd1a80

Reply:
I admit Apple documentation is confusing. So I remade the IBOutlets from XCode just to confirm.


https://github.com/xmartlabs/XLPagerTabStrip/assets/1507391/8b55e26c-e020-458f-9810-fc68a3de9816

Reply:
I have seen both happen by default. For example when you build a UIViewController and want to connect its `view` to the controller class it defaults to strong (as that `view` should be strong). But for other views which are subviews of the main `view` it will default to weak (because subviews will be strongly referenced by their parent). This means as long as a view is in the view hierarchy it should not be deallocated and you should have no issue. However if you plan to remove the view you will have to strongly reference it if you plan to continue using it afterwards.

I think it is best to revert unless there are reproducible crashes which I would rather think could be linked to how the views are used. It might make sense to switch some of the outlets to strong if the use cases require it (rather than just switching everything)